index.test-d.ts

```diff
@@ -0,0 +1,58 @@
+import { expectType } from 'tsd';
+import axios, { AxiosError, AxiosRequestConfig } from 'axios';
+import axiosRetry, {
+  IAxiosRetryConfig,
+  IAxiosRetryConfigExtended,
+  IAxiosRetryReturn,
+  exponentialDelay,
+  isIdempotentRequestError,
+  isNetworkError,
+  isNetworkOrIdempotentRequestError,
+  isRetryableError,
+  isSafeRequestError
+} from './index.js';
```

Comment:
I confirm that `name import` is working here (and that axiosRetry is able to import default).